onnx/defs/sequence/defs.cc

```diff
@@ -634,4 +634,297 @@ ONNX_OPERATOR_SET_SCHEMA(
           }
         }));
 
+static const char* SequenceMap_ver16_doc = R"DOC(
+Applies a sub-graph to each sample in the input sequence(s).
+
+Inputs can be either tensors or sequences, with the exception of the first input which must
```

Comment:
Is the idea to support a form of broadcast (if one of the inputs is not a sequence)?

Reply:
Yes, that was my idea

Reply:
I assume that all sequences are expected to have the same length? It would be useful to explain that constraint here.